### PR TITLE
Add NFT initial types/interfaces

### PR DESCRIPTION
**Pull request by holygits**

### CODE REVIEW

File: packages/types/src/interfaces/grandpa/definitions.ts [hunk 1]
```diff
@@ -1,10 +0,0 @@
-export default {
-  rpc: {
-    restartVoter: {
```
Comment (KarishmaBothara) [inline packages/types/src/interfaces/grandpa/definitions.ts:3]: I think we need this rpc call? else we will get warning for grandpa.rpc.restartVoter does not exist?

**Comment by holygits** [comment]
@KarishmaBothara I will split this PR into the clean up, update, and NFT module stuff

File: packages/types/src/interfaces/grandpa/definitions.ts (hunk 1 above)
Comment (holygits) [reply packages/types/src/interfaces/grandpa/definitions.ts:3]: yeah, wasn't sure how to make it merge with the substrate definitions

File: packages/api/src/derives/staking/index.ts [hunk 2]
```diff
@@ -1,6 +1,5 @@
 export { queryStakingAccountInfo as accountInfo } from './stakingAccount';
 export * from './overview';
-export * from './query';
```
Comment (holygits) [inline packages/api/src/derives/staking/index.ts:3]: doesn't build dues to type issues, it isn't used from what I can see so opting to remove

File: tsconfig.json [hunk 3]
```diff
@@ -22,7 +22,9 @@
       "@cennznet/types/*": ["packages/types/src/*"],
       "@cennznet/types": ["packages/types/src/index"],
       "@cennznet/util/*": ["packages/util/src/*"],
-      "@cennznet/util": ["packages/util/src/index"]
+      "@cennznet/util": ["packages/util/src/index"],
+      "@polkadot/api/augment": ["packages/types/src/interfaces/augment-api.ts"],
```
Comment (holygits) [inline tsconfig.json:26]: fixes the `//@ts-ignores` in augment-types

File: packages/api/src/derives/staking/index.ts (hunk 2 above)
Comment (KarishmaBothara) [reply packages/api/src/derives/staking/index.ts:3]: added this so that we can remove decorating api with staking derived queries in apps/UI project..

**Review by KarishmaBothara — APPROVED** [review]

File: packages/api/src/derives/staking/index.ts (hunk 2 above)
Comment (holygits) [reply packages/api/src/derives/staking/index.ts:3]: ah so will this break the UI?

Comment (KarishmaBothara) [reply packages/api/src/derives/staking/index.ts:3]: no.. it won't... the UI is decorating it atm.

**Pull request merged**